.circleci/config.yml

```diff
@@ -378,6 +378,12 @@ jobs:
               -e "ZONE_NAME=${ZONE_NAME}" \
               -e "ENV=${ENV}" \
               --rm efcms /bin/sh -c "./setup-for-blue-green-migration.sh"
+      - run:
+          name: Setup Migrate Flag
+          command: echo "export MIGRATE_FLAG=$(./scripts/get-migrate-flag.sh $ENV)" >> $BASH_ENV
+      - run:
+          name: Setup Destination Table
+          command: echo "export DESTINATION_TABLE=$(./scripts/get-destination-table.sh $ENV)" >> $BASH_ENV
```

Comment:
Thanks for the fix! I see why this is needed.

I’m a bit worried that we’ll remove these in the future by accident, but that’s just speculation at this point.